### DISCUSSION TITLE
Migration from Strapi

### DISCUSSION THREAD

**Discussion by vmloskutov** [directus/directus#9835] [Archive]
Hi, is there any mechanism to migrate from Strapi to DirectUs?

**Comment by benhaynes** [comment]
No custom scripts, but if you can get your data into SQL, CSV, or other DB friendly format, then you can simply install Directus on top of that to manage it. 👍

**Comment by johnstyles940** [comment]
Hi good day, understand that this ticket has closed, but may i know if we can install directus, and point to the existing Strapi's database?

**Reply by hugopvl** [reply]
what you can do is install directus directly on top of the strapi database, you won't lose the data, directus will just add new tables, once installed you can manage all of them. this if the strapi db is in sql or mysql, or some type compatible with directus https://directus.io/#databases
